tensorflow/lite/micro/kernels/xtensa/lstm/helper_utils/portable_tensor_utils.cc

```diff
@@ -0,0 +1,775 @@
+/* Copyright 2021 The TensorFlow Authors. All Rights Reserved.
```

Comment:
Is this a copy of an existing file in the tree? It's hard to tell what's different without a diff view.